File: Iniciante/1010/1010.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Arrays;

 
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        
        Scanner nabana = new Scanner(System.in);


        double[] n1 = Arrays.stream(nabana.nextLine().split(" "))
                .mapToDouble(Double::parseDouble)
                .toArray();
        double[] n2 = Arrays.stream(nabana.nextLine().split(" "))
                .mapToDouble(Double::parseDouble)
                .toArray();
        System.out.println("VALOR A PAGAR: R$ " + String.format("%.2f", (n1[1]*n1[2])+(n2[1]*n2[2])));
    }

File: Iniciante/1010/1010.cs
using System; 

class URI {

    static void Main(string[] args) { 

 string[] numeros_1 = Console.ReadLine().Split(' ');
int A = int.Parse(numeros_1[0]);
int B = int.Parse(numeros_1[1]);
double C = double.Parse(numeros_1[2]);


string[] numeros_2 = Console.ReadLine().Split(' ');
int D = int.Parse(numeros_2[0]);
int E = int.Parse(numeros_2[1]);
double F = double.Parse(numeros_2[2]);

Console.WriteLine("VALOR A PAGAR: R$ {0:f2}", (B * C + E * F));

    }

}